src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -19,12 +19,14 @@ To check if you have access to these features, you can go to the [authentication
 
 Requirements to configure these settings:
 
-* These features are for managing users on the [New Relic One user model](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles). For users on our original user model, see [Original account management](/docs/accounts/original-accounts-billing).
-* Configuring these settings requires [Pro or Enterprise edition](https://newrelic.com/pricing). 
-* To edit these settings, you must be in a group with the [**Authentication domain manager** role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles).
-* [SCIM provisioning](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign), also known as automated user management requires Pro or Enterprise edition.
-* SAML SSO requires Pro or Enterprise edition. SAML support includes:
-  * [Active Directory Federation Services (ADFS)](http://technet.microsoft.com/en-us/library/hh831502.aspx "Link opens in a new window")
+* Authentication domains are for managing users on the [New Relic One user model](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles). For users on our original user model, see [Original account management](/docs/accounts/original-accounts-billing).
```

Comment:
Can we delete the second sentence? the important call out makes this distinction, so you might not need it here, too. 

Also, since this bullet point defines that authentication is for New Relic One user models (and the whole page is about authentication, not just the requirements section), I might move this sentence into the intro portion for context-giving.